akka-projection-cassandra/src/test/scala/akka/projection/cassandra/CassandraProjectionSpec.scala

```diff
@@ -407,15 +407,15 @@ class CassandraProjectionSpec
         sinkProbe.request(1000)
 
         (1 to 15).foreach { n =>
-          sourceProbe.get.sendNext(Envelope(entityId, n, s"elem-$n"))
+          sourceProbe.get.sendNext(Envelope(entityId, n.toLong, s"elem-$n"))
         }
         eventually {
           repository.findById(entityId).futureValue.get.text should include("elem-15")
         }
         offsetStore.readOffset[Long](projectionId).futureValue.get shouldBe 10L
 
         (16 to 17).foreach { n =>
-          sourceProbe.get.sendNext(Envelope(entityId, n, s"elem-$n"))
+          sourceProbe.get.sendNext(Envelope(entityId, n.toLong, s"elem-$n"))
```

Comment:
I think we should use `-Ywarn-numeric-widen`. This warning makes no sense to me.

Reply:
You mean that we should not use it? 

True that going from Int to Long should not be a problem.

Reply:
btw, this is what is used in akka/akka. Although, most projects are on this `loose` list so maybe that's why we are not being annoyed by it there.

Reply:
yes, I mean "not use". A few days ago I suggested in some PR that we should remove the `loose` and use that for all projects since it seems like all are added to that list anyway